src/providers/pkcs11/mod.rs

```diff
@@ -225,6 +225,20 @@ impl Provide for Provider {
         })
     }
 
+    fn list_clients(&self, _op: list_clients::Operation) -> Result<list_clients::Result> {
+        let store_handle = self.key_info_store.read().expect("Key store lock poisoned");
+        Ok(list_clients::Result {
+            clients: key_info_managers::list_clients(store_handle.deref(), ProviderID::MbedCrypto)
+                .map_err(|e| {
+                    format_error!("Error occurred when fetching key information", e);
+                    ResponseStatus::KeyInfoManagerError
+                })?
+                .into_iter()
+                .map(|e| e.to_string())
```

Comment:
Does `e` mean `client`?

Reply:
I always use `e` as Element but could also be cliEnt, right :shipit: ? I can change to `client`

Reply:
`app_name` is more correct actually

Reply:
Yeah, I was confused because I tend to use `e` only when it's an `error` :joy: